oop_verify.py

```diff
@@ -124,14 +124,9 @@ class ProjectFilesVerifier(Verifier):
     def __init__(self):
         Verifier.__init__(self)
         self.pages = []
-        self.minimum_size = 410
+        self.minimum_size = 380
         self.page_elements = [
```

Comment:
Are you not gunna add these elements to self.page_elements in the Verifier super class? Do we not need them?